### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by rullzer** [nextcloud/files_antivirus#63]
It probably makes sense to add notifications when a virus is detected (at least when performing a background scan).

1. Send notification for the user owning the file
2. Send notification to admin

I guess this is now days better than e-mail since we have push notifications anyway. And you could argue that this kind of stuff warrants a notification since some action is desired I'd say.

@nickvergessen for input if this makes sense?

**Comment by nickvergessen** [comment]
Yeah, basically the code from ransomware_protection could be copied

**Comment by Ichigo103** [comment]
Hi !!
Did you find a solution ?

**Comment by rullzer** [comment]
Yes. I know the solution. I just need time to actually do it ;)

**Comment by PhantomPhreak** [comment]
Would like to have this feature as well!

**Comment by itverwaltung** [comment]
I suggest to add the option to have notifications within the activities menu item. This would really be great. For the moment i tinker on some code greping the log file for "Infected"-String and generate an e-mail ... not very nice, right?
Thank you for your support!

**Comment by mikeylikesrocks** [comment]
Would be a great improvement to have an email notification sent when a file is marked as infected.

**Comment by MaxMcBurn** [comment]
yes please implement it .. thats a great improvement
+1

**Comment by julian70400** [comment]
Is this planned ? A simple mail to all admin would be enough..

**Comment by ShapeShifter499** [comment]
I'd love to have the option for an email notification to be sent when a virus is detected including which file it is.
